modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -0,0 +1,119 @@
+// This is included in the following assemblies:
+//
+// ipi-install-expanding-the-cluster.adoc
+
+[id='preparing-the-bare-metal-node_{context}']
+
+= Preparing the bare metal node
+
+Preparing the bare metal node requires executing the following procedure from the provisioner node or from a control plane node.
+
+.Procedure
+
+. Get the `oc` binary, if needed. It should already exist on the provisioner node.
++
+----
+[kni@openshift-master-0 ~]$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/openshift-client-linux.tar.gz | tar zxvf - oc
+[kni@openshift-master-0 ~]$ sudo cp oc /usr/local/bin
```

Comment:
Separate multiple commands out into their own command blocks per our updated style guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output